src/writeFlashcardsOllama.ts

```diff
@@ -0,0 +1,26 @@
+import ollama from "ollama";
+import {
+  readFileContent,
+  writeToJsonFile,
+  convertToAnkiDeck,
+  writeToTextFile,
+} from "./utils/fileOperations";
+
+async function writeFlashcardsOllama(model: string, inputTextPath: string) {
+  const instructions = await readFileContent("src/prompts/prompt_json.txt");
+  const textContext = await readFileContent(inputTextPath);
+
+  const response = await ollama.chat({
+    model: model,
+    messages: [{ role: "user", content: instructions + textContext }],
+  });
+
+  writeToJsonFile("src/responses/response", response.message.content).then(
+    async () => {
+      const ankiDeck = await convertToAnkiDeck("src/responses/response.json");
+      writeToTextFile("src/responses/response", ankiDeck);
+    }
+  );
+}
```

Comment:
I don't think using a path to a file is a future-proof solution. If we change `inputTextPath` to a mere text, we can use this function to parse text files, presentations, and any other files we may think of – provided we create functions to extract text from them and pass the extracted text to this function.

Reply:
Okay, I had a different idea for this, but that what you said is probably better. I'll change it.